### PR TITLE
[zh] Resync server-side-apply page

### PR DESCRIPTION
**Pull request by tengqm**
Related: #27673

### CODE REVIEW

File: content/zh/docs/reference/using-api/server-side-apply.md [hunk 1]
```diff
@@ -399,18 +397,116 @@ Kubernetes 1.16 和 1.17 中添加了一些标记，
...
-| `//+listMapKey` | `x-kubernetes-list-map-keys` | Slice of map keys that uniquely identify entries for example `["port", "protocol"]` | Only applicable when `+listType=map`. A slice of strings whose values in combination must uniquely identify list entries. While there can be multiple keys, `listMapKey` is singular because keys need to be specified individually in the Go type. | 1.16 |
+| `//+listType` | `x-kubernetes-list-type` | `atomic`/`set`/`map` | Applicable to lists. `set` applies to lists taht include only scalar elements. These elements must be unique. `map` applies to lists of nested types only. The key values (see `listMapKey`) must be unique in the list. `atomic` can apply to any list. If configured as `atomic`, the entire list is replaced during merge. At any point in time, a single manager owns the list. If `set` or `map`, different managers can manage entries separately. | 1.16          |
```
Comment (howieyuen) [inline content/zh/docs/reference/using-api/server-side-apply.md:400]: taht? that

File: content/zh/docs/reference/using-api/server-side-apply.md [hunk 2]
```diff
@@ -399,18 +397,116 @@ Kubernetes 1.16 和 1.17 中添加了一些标记，
...
 | `//+structType` | `x-kubernetes-map-type` | `atomic`/`granular` | Applicable to structs; otherwise same usage and OpenAPI annotation as `//+mapType`.| 1.17 |
 -->
 | Golang 标记 | OpenAPI extension | 可接受的值 | 描述 | 引入版本 |
 |---|---|---|---|---|
-| `//+listType` | `x-kubernetes-list-type` | `atomic`/`set`/`map` | 适用于 list。 `atomic` 和 `set` 适用于只包含标量元素的 list。 `map` 适用于只包含嵌套类型的 list。 如果配置为 `atomic`, 合并时整个列表会被替换掉; 任何时候，唯一的管理器都把列表作为一个整体来管理。如果是 `set` 或 `map` ，不同的管理器也可以分开管理条目。 | 1.16          |
-| `//+listMapKey` | `x-kubernetes-list-map-keys` | 用来唯一标识条目的 map keys 切片，例如 `["port", "protocol"]` | 仅当 `+listType=map` 时适用。组合值的字符串切片必须唯一标识列表中的条目。尽管有多个 key，`listMapKey` 是单数的，这是因为 key 需要在 Go 类型中单独的指定。 | 1.16 |
+| `//+listType` | `x-kubernetes-list-type` | `atomic`/`set`/`map` | 适用于 list。`set` 适用于仅包含标量元素的列表。这些元素必须是不重复的。`map` 仅适用于包含嵌套类型的列表。列表中的键（参见 `listMapKey`）不可以重复。`atomic` 适用于任何类型的列表。如果配置为 `atomic`，则合并时整个列表会被替换掉。任何时候，只有一个管理器负责管理指定列表。如果配置为 `set` 或 `map`，不同的管理器也可以分开管理条目。 | 1.16          |
+| `//+listMapKey` | `x-kubernetes-list-map-keys` | 字段名称的列表，例如，`["port", "protocol"]` | 仅当 `+listType=map` 时适用。取值为字段名称的列表，这些字段值的组合能够唯一标识列表中的条目。尽管可以存在多个键，`listMapKey` 是单数的，这是因为键名需要在 Go 类型中各自独立指定。 | 1.16 |
```
Comment (howieyuen) [inline content/zh/docs/reference/using-api/server-side-apply.md:408]: 第三列的最后一句话缺失：“The key fields must be scalars”，键字段必须是标量。

**Review by howieyuen — CHANGES_REQUESTED** [review]

File: content/zh/docs/reference/using-api/server-side-apply.md (hunk 2 above)
Comment (tengqm) [reply content/zh/docs/reference/using-api/server-side-apply.md:408]: done

**Review by chenrui333 — APPROVED** [review]
/lgtm
/approve

**Pull request merged**